File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        vector<int> newArray;

        newArray.push_back(nums[0]);

        for(int i=1; i<nums.size(); i++) {
            if(nums[i-1] != nums[i]) {
                newArray.push_back(nums[i]);
            }
        }

        for(int i=0; i<newArray.size(); i++) {
            nums[i] = newArray[i];
        }

        return newArray.size();
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_count = 0
        counter = 0;

        for num in nums:
            if num == 1:
                counter += 1
                max_count = max(max_count, counter)
            else:
                counter = 0
        
        return max_count


